distanceDiracGaussian.m

```diff
@@ -0,0 +1,83 @@
+function [D,G]=distanceDiracGaussian(x,N,L)
+%% 
+% distanceDiracGaussian.m calculates the distance between dirac mixture
+% approximation and normal distribution in terms of Localized Cumulative
+% Distribution.
+%
+% Parameters:
+%   x: location of dirac distributions
+%   N: dimension of dirac mixture approximation
+%   L: number of component for each dimension 
+%
+% Return values:
+%   D: distance between dirac mixture approximation and normal distribution
+%   G: gradient of D
+%
+% History:
+% * 2018/01/04 Dantong Wang
+
```

Comment:
Maybe some short comment, how the whole distance is computed (which summands and integrals are needed) would be helpful here. This would give more meaning to the abbreviations D1, D2, D3 etc. for other developers. Alternatively (or additionally), reference the paper here using title and author.